app/controllers/health/careplans_controller.rb

```diff
@@ -55,25 +55,39 @@ def index
     end
 
     def show
+      @pdf = true
+      @html = false
+      # if careplan is editable, we need these variables used by list partials
+      if @careplan.editable?
+        @services = @patient.services
+        @equipments = @patient.equipments
+        @backup_plans = @careplan.backup_plans
+      end
+
       pdf = careplan_combine_pdf_object
-      file_name = 'care_plan'
+      file_name = "care_plan_#{DateTime.current.to_s(:db)}"
```

Comment:
Do we, maybe want to use the updated date or created date for the file name?  It seems a bit odd you might download the same file and get different file names, but I do appreciate that they won't all be called "care_plan".  Maybe we should add the client or patient ID too?  I'm hesitant to include the patient name, but that would probably be most useful.

Reply:
I don't think patient name is a good idea. Unless we hear a compelling reason to do that. I'll change it to the updated date!